image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml

```diff
@@ -92,6 +92,14 @@ spec:
             drop: ["NET_RAW"]
           readOnlyRootFilesystem: true
         env:
+        - name: GOMEMLIMIT
```

Comment:
Is rocksdb included in this limit?

Reply:
RocksDB will no longer be in the image for 4.0 so this will not be as important

Reply:
but also in general, this is only for memory that go knows about so it won't be as effective on rocksdb